### PR TITLE
Feature/andrew

### PR DESCRIPTION
**Pull request by ahnilica**

### CODE REVIEW

File: src/app/containers/Facilities/All/index.jsx [hunk 1]
```diff
@@ -132,15 +174,14 @@ const AllFacilities = () => {
               <Div>
                 <AdaptiveGrid defaultItemsPerRow={1} breakpoints={{ "992": 2 }}>
                   {/* Map through facilities and display facility cards */}
-                  { facilities?.map((facility) => <FacilityCard key={facility?.provider_id} facility={facility} /> )}
+                  { facilities?.map((facility, index) => <FacilityCard key={index} facility={facility} /> )}
```
Comment (mmnavarr) [inline src/app/containers/Facilities/All/index.jsx:177]: Keys are supposed to be unique and not based off the index - why the change?

Comment (ahnilica) [reply src/app/containers/Facilities/All/index.jsx:177]: I believe I ran into a key related error, not sure why but I didn't give it much thought.
Perhaps bad data?
